packages/document/main-doc/docs/zh/guides/basic-features/deploy.mdx

```diff
@@ -131,30 +134,32 @@ Netlify 是一个流行的 web 开发平台，专为构建、发布和维护现
 
 ### Monorepo 项目
 
-对于 Monorepo 项目，除了需要构建我们当前的项目外，还需要构建当前项目的依赖的其他仓库。我们以一个 pnpm monorepo 仓库为例，
-在 Netlify 上对 Monorepo 项目进行部署。
+对于 Monorepo 项目，除了需要构建当前的项目外，还需要构建当前项目依赖的仓库中其他子项目。这里以一个 pnpm Monorepo 仓库为例，在 Netlify 上对 Monorepo 项目进行部署。
 
-假设我们有以下 Monorepo 仓库：
+假设 Monorepo 仓库目录结构如下：
 
 ```
-./
-├── packages/
-│   ├── app/
+.
+├── packages
+│   ├── app
 │   └── app-dep1
 ├── package.json
 ├── pnpm-lock.yaml
 └── pnpm-workspace.yaml
 ```
 
-我们需要在 netlify 平台上配置 Base directory 为 `packages/app`:
+你需要在 Netlify 平台上配置 **Base directory** 为 `packages/app`:
 
 <img src="https://sf16-sg.tiktokcdn.com/obj/eden-sg/lmeh7nuptpfnuhd/netlify-monorepo-basedir.png?x-resource-account=public" />
 
 在 `packages/app/package.json` 中添加以下 script，在执行 `app` 仓库的部署命令之前，先执行 workspace 中其他仓库的构建：
 
 ```json
-"scripts": {
-  "deploy:app": "pnpm --filter 'app^...' run build && pnpm --filter=app run deploy",
+{
+  "scripts": {
```

Comment:
The deployment command has been modified here, but the description of the `netlify.toml` file has not been modified, which will lead to failed deployment

Reply:
OK let me fix it..

Reply:
done